Makefile

```diff
@@ -393,7 +393,7 @@ run: generate fmt vet ## Run a controller from your host.
 	go run ./main.go
 
 docker-build: ## Build docker image with the manager.
-	docker build -t $(IMG) .
+	docker build -t $(IMG) .  --load
```

Comment:
If using podman runtime with docker cli also, there was the following warning locally for me:
```
 => CACHED [stage-1 2/3] COPY --from=builder /workspace/manager .                                                                                  0.0s
WARNING: No output specified with docker-container driver. Build result will only remain in the build cache. To push result image into registry use --push or to load image into docker use --load
```
This can cause the cluster to use an old build image as the newly built image is kept in cache only